ui/decredmaterial/clickable_list.go

```diff
@@ -53,6 +59,11 @@ func (cl *ClickableList) handleClickables(count int) {
 func (cl *ClickableList) Layout(gtx layout.Context, count int, w layout.ListElement) layout.Dimensions {
 	cl.handleClickables(count)
 	return cl.List.Layout(gtx, count, func(gtx C, i int) D {
+		if cl.IsShadowEnabled && cl.clickables[i].button.Hovered() {
+			return cl.shadow.Layout(gtx, func(gtx C) D {
+				return cl.row(gtx, count, i, w)
+			})
+		}
```

Comment:
You don't need to store a `cl.shadow` property.
```suggestion
		if cl.IsShadowEnabled && cl.clickables[i].button.Hovered() {
			shadow = cl.theme.Shadow()
			shadow.SetShadowRadius(14)
			shadow.SetShadowElevation(5)
			return shadow.Layout(gtx, func(gtx C) D {
				return cl.row(gtx, count, i, w)
			})
		}
```